nutri-track-bot/src/main/java/com/nutritrack/bot/service/telegram/factory/TelegramFactory.java

```diff
@@ -29,11 +29,15 @@ public static TelegramWebhookBot webhookBot(String token,
         return new TelegramWebhookBot(token) {
 
             {
-                registerCommands();
+                registerCommands("en");
```

Comment:
1. This code is called just once, when the app is launching. So when the app gets a real request, these methods will not be launched.
2. You can get the language code from the Local object, so you don't need constants in this case. See example in this class: com.nutritrack.bot.config.MessageConfig
3. In other cases, if you need constant variables, please move them into an Enum class and use them across your code from that one place.